lua/core/utils/init.lua

```diff
@@ -282,37 +229,35 @@ end
 -- @param module the module path of the override setting
 -- @param default the default settings that will be overridden
 -- @param extend boolean value to either extend the default settings or overwrite them with the user settings entirely (default: true)
--- @param prefix a module prefix for where to search (default: user)
 -- @return the new configuration settings with the user overrides applied
-function astronvim.user_plugin_opts(module, default, extend, prefix)
+function astronvim.user_opts(module, default, extend)
   -- default to extend = true
```

Comment:
Is there any way you could keep `prefix` as an input parameter? I find this function really useful, if not, then I can copy it over

Reply:
Sure, what do you use the function for? I could make it so the prefix is default not `user` maybe. How are you using it in your configuration?

Reply:
@jay-babu I was looking through it and I'm not sure how this function is useful with the prefix? All it does is change whether or not it's in the `user` module. And in your user configuration you should always be getting from the `user` module so I don't think the prefix is necessary. You can just add your prefix to the `module` parameter

Reply:
That makes sense. I will attempt to do that